docs/flink-ddl.md

```diff
@@ -169,13 +184,15 @@ To create a partition table, use `PARTITIONED BY`:
 ```sql
 CREATE TABLE `hive_catalog`.`default`.`sample` (
     id BIGINT COMMENT 'unique id',
-    data STRING
-) PARTITIONED BY (data);
+    `data` STRING NOT NULL
+) 
+PARTITIONED BY (data) 
+WITH ('format-version'='2');
 ```
 
-Iceberg support hidden partition but Flink don't support partitioning by a function on columns, so there is no way to support hidden partition in Flink DDL.
+Iceberg supports hidden partition but Flink doesn't support partitioning by a function on columns, so there is no way to support hidden partition in Flink DDL.
 
-### `CREATE TABLE LIKE`
```

Comment:
Same comment here for below. Let's keep this focused on fixing these descriptions and do this in a larger PR.